src/lib.rs

```diff
@@ -269,6 +269,7 @@ pub enum BuiltIn {
     // fragment
     FragDepth,
     FrontFacing,
+    PrimitiveId,
```

Comment:
the current naming convention for builtins is such that "XxxID" are vectors, and all the scalars are "XxxIndex".
So let's call this `PrimitiveIndex`.